commands/cluster-nodes.md

```diff
@@ -17,31 +17,31 @@ It is also used by `redis-cli` in order to manage a cluster.
 ## Serialization format
 
 The output of the command is just a space-separated CSV string, where
-each line represents a node in the cluster. Starting from 7.2.0, the
-output of the command always contains a new auxiliary field called
-shard-id. The following is an example of output on Redis 7.2.0.
+each line represents a node in the cluster. The auxiliary fields 
+(including shard ID, human node name and extra port number) stored in
+nodes.conf file are currently hidden from client commands. The following
```

Comment:
```suggestion
each line represents a node in the cluster. The following
```

Our historical guidance has been the command should show the latest state, since aux fields aren't the latest we can leave them omitted.